RFCs/platform-practice-4ps.md

```diff
@@ -0,0 +1,82 @@
+---
+name: Platform Practice 4Ps
+about: With this document, I am proposing a review of the platform practice using the 4Ps for meetings(Purpose, Product, Process and People)
+
+---
+
+<!-- Read the docs about [how to write an RFC at Artsy](https://github.com/artsy/README/blob/43c400d81ff9fee7276c3dd934de26b985da362f/playbooks/rfcs.md) before starting an RFC.
+ -->
+ 
+# Platform Practice
+
+Using the 4Ps format to organize our thoughts around this practice
+
+## Purpose:
+
+-   Disseminate good patterns: Share best practices and successful patterns within the platform development.
+    
+-   Actions to improve the platform: Identify and discuss actionable items to enhance the platform.
+    
+-   Avenue for people to bring what they find important: Provide a platform for individuals to contribute their insights and concerns.
+    
+-   Radiate intent of changes in the Platform: Communicate the purpose and intent behind any changes.
+    
+-   Evolving a platform towards a coherent direction: Ensure alignment among engineers and product teams, steering the platform in a cohesive direction.
+    
+
+## Product:
+
+*   Practice notes with the main topics: Document key points and decisions made during the practice.
+*   Possible action items discussed: Identify and outline potential actions for improvement.
+*   Information was shared and people aligned: Ensure information is disseminated and team members are aligned on the discussed topics.
+    
+
+## People:
+
+* It's open to everybody in PDDE and part of the engineering team calendar: Encourage inclusivity by making the practice accessible to all team members.
+
+### Roles of this practice:
+
+#### 1. Facilitator:
+   * The Facilitator plays a crucial role in managing the process of the platform practice. This includes time-keeping, organizing the agenda, documenting decisions, and following up on action items. The Facilitator ensures the flow and effectiveness of each session.
+
+#### 2. Champion:
+   * The Champion is the driving force behind topics discussed in the platform practice. They bring forth specific subjects with clear objectives, whether seeking a decision or fostering an open discussion. The Champion actively guides discussions, aiding the Facilitator in achieving meaningful conclusions for each topic.
+
+#### 3. Curator:
+   * The Curator takes on the responsibility of shaping the overall direction of the platform practice. They provide valuable advice and curate relevant topics that align with the practice's goals. The Curator's role is crucial in maintaining the practice's focus and ensuring that discussions contribute to the overall improvement and coherence of the platform.
+
+
+## Process:
```

Comment:
Something I appreciated recently was reminding engineers that the meeting is happening in X minutes with a clear agenda
<img width="1239" alt="Screenshot 2024-02-13 at 11 20 57" src="https://github.com/artsy/README/assets/11945712/01498c84-9d33-490c-84cd-a4437d77f705">